src/Stratis.Bitcoin.Features.RPC/RPCMiddleware.cs

```diff
@@ -54,6 +55,12 @@ public RPCMiddleware(RequestDelegate next, IRPCAuthorization authorization, ILog
                 httpContext.Response.ContentType = "application/json";
                 await httpContext.Response.WriteAsync(response.ToString(Formatting.Indented));
             }
+            else if (ex is ConfigurationException)
```

Comment:
A suggested improvement is to use a switch case on each type (like we do in some of the behaviours)

Reply:
+1

Reply:
It especially doesn't look nice when a lot of code is repeated

why don't We have
```
httpContext.Response.ContentType = "application/json";
await httpContext.Response.WriteAsync(response.ToString(Formatting.Indented));
```
These 2 guys just once?

Reply:
I agree. But as far as the switch is concerned, I don't think it's an improvement.

Reply:
It should be catch clause not switch anyway, but whatever, feel free to leave it for this PR